scripts/coverage.sh

```diff
@@ -21,7 +21,7 @@
 #
 ###########################################################################
 
-autoreconf -fi
+/usr/bin/autoreconf -fi
```

Comment:
Why should this one use an absolute path when no others do?

Reply:
revet it here: [revert: absolute path and whitespace](https://github.com/curl/curl/pull/8777/commits/ff663e5a652f0b28c9292f8642c01cbcaa2204f4)
thought it would be useful in a bash script